src/components/Layout/HomeContent.js

```diff
@@ -160,7 +164,9 @@ export function HomeContent() {
 
         <Section background="left-card">
           <Center>
-            <Header>컴포넌트를 사용하여 사용자 인터페이스 만들기</Header>
+            <Header>
+              컴포넌트를 사용하여 <Br /> 사용자 인터페이스 만들기
```

Comment:
```suggestion
              컴포넌트를 사용하여
              <Br /> 사용자 인터페이스 만들기
```

얘도 일관성을 맞춰 수정하는게 좋을 듯 합니다